### DISCUSSION TITLE
Portable version?

### DISCUSSION THREAD

**Discussion by Tom4tot** [complexlogic/EasyAudioSync#3] [General]
Hello,

First, a small question; would it possible that you share a Windows portable version of this program? 

But mostly: it's been months I'm doing things manually and it was a really pita, this program is really changing my life! Thanks so much. Great simple UI, amazing performance, has all the settings I need. And clearly the easiest program to use.

**Comment by complexlogic** [comment]
Glad to hear the program is working well for you.

The executable should have no issue running in portable manner. The only issue is the application settings. Those are currently stored in the Windows registry, which is obviously not portable. Qt does support storing the settings as an .ini file (the Linux version already uses this). Perhaps this would be feasible for a Windows portable version.

I'll consider it for a future version of the program.
